### PR TITLE
updates to MR1 dashboard

### PR DESCRIPTION
**Pull request by SuYoungHong**
Update to production MR1 Dashboard

### CODE REVIEW

**Comment by SuYoungHong** [comment]
I've updated all the underlying files, and created a dashboard file... but the dashboard seems to be throwing error now...

File: duet/dashboards/firefox_mr1_launch.dashboard.lookml [hunk 1]
```diff
@@ -0,0 +1,2932 @@
+- dashboard: firefox_mr1_launch
+  title: Firefox MR1 Launch
+  layout: newspaper
+  preferred_viewer: dashboards-next
+  load_configuration: wait
+  refresh: 1 day
+  elements:
+  - title: Uplift Desktop
+    name: Uplift Desktop
+    model: DUET
```
Comment (scholtzan) [inline duet/dashboards/firefox_mr1_launch.dashboard.lookml:10]: Tested on a fork.
```
    model: duet
```
also needs to be changed for all other occurrences

File: duet/dashboards/firefox_mr1_launch.dashboard.lookml [hunk 2]
```diff
@@ -0,0 +1,2932 @@
...
+    height: 2
+  - name: ''
+    type: text
+    title_text: ''
+    subtitle_text: ''
+    body_text: |-
+      -------------------------
+      # Upgrade Adoption
+
+      Release Date: 2021-06-01
```
Comment (scholtzan) [inline duet/dashboards/firefox_mr1_launch.dashboard.lookml:100]: Should `# Upgrade Adoption` be the `title_text`? It looks like Looker doesn't renders the `body_text`, but instead just shows the raw text

File: duet/dashboards/firefox_mr1_launch.dashboard.lookml (hunk 1 above)
Comment (SuYoungHong) [reply duet/dashboards/firefox_mr1_launch.dashboard.lookml]: updated and confirmed

File: duet/dashboards/firefox_mr1_launch.dashboard.lookml (hunk 2 above)
Comment (SuYoungHong) [reply duet/dashboards/firefox_mr1_launch.dashboard.lookml:104]: putting it in body text was by design ... so the markdown is rendering in some spots, but not in all of them,

**Review by scholtzan — APPROVED** [review]

**Pull request merged**